src/typings.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Discord ID
  */
-export type Snowflake = string;
```

Comment:
Isn't this technically a breaking change?

Reply:
The difference between `string` and `${bigint}` is negligible, it doesn't actually change anything, it's just more descriptive as to what a Snowflake actually is

Reply:
I think it's more about the users who already use typescript and strings for IDs in their setup. It might error out if the type no longer is the same as the meta-type, causing them to import and refactor strings to Snowflakes.

Reply:
> I think it's more about the users who already use typescript and strings for IDs in their setup. It might error out if the type no longer is the same as the meta-type, causing them to import and refactor strings to Snowflakes.

You could technically argue it's breaking since it would fail at compile time that it's an error, however, anything that's a string with only numbers would still pass the `${bigint}` check. And if they had a string that wasn't only numbers it'd error anyways at the API call no?

Reply:
![image](https://user-images.githubusercontent.com/7953076/124168582-59f11c80-da73-11eb-985b-a6cb362703fa.png)